test/parallel_api/iterator/permutation_iterator_parallel_transform_reduce.pass.cpp

```diff
@@ -56,7 +56,7 @@ DEFINE_TEST_PERM_IT(test_transform_reduce, PermItIndexTag)
                     dpl::copy(exec, permItBegin, permItEnd, sourceData.begin());
                     wait_and_throw(exec);
 
-                    const auto expected = ::std::transform_reduce(sourceData.begin(), sourceData.end(), TestValueType{},
+                    const auto expected = transform_reduce_serial(sourceData.begin(), sourceData.end(), TestValueType{},
```

Comment:
Should not we whether `TestUtils` namespace for 'transform_reduce_serial'?

Reply:
Yes, we should use it.  Fixed.

Reply:
Technically, as far as we have `using namespace TestUtils;` in the file `permutation_iterator_common.h`
this namespace specification isn't required.